tests/materials/linear_elastic_test.cc

```diff
@@ -147,6 +145,22 @@ TEST_CASE("LinearElastic is checked in 2D", "[material][linear_elastic][2D]") {
     REQUIRE(stress(4) == Approx(0.00000000000000e+00).epsilon(Tolerance));
     REQUIRE(stress(5) == Approx(0.00000000000000e+00).epsilon(Tolerance));
   }
+
+  SECTION("LinearElastic check properties earthquake") {
+    unsigned id = 0;
+    jmaterial["p_wave_velocity"] = 116.023870223;
```

Comment:
This is an incorrect text, you can't pass the values of p_wave and s_wave velocity and check if that's correct.